doc/nrf/protocols/bt_mesh/configuring.rst

```diff
@@ -35,11 +35,113 @@ Mesh models
 
+Mesh settings/performance
+=========================
+
+More information about mesh memory footprint, see :ref:`app_memory_bt_mesh`.
+
+* :kconfig:option:`CONFIG_BT_MESH_PROXY_USE_DEVICE_NAME` - Includes GAP device name in scan response when the GATT Proxy feature is enabled.
+* :kconfig:option:`CONFIG_BT_MESH_DK_PROV` - Bluetooth mesh provisioning handler for the nRF5x development kits.
+* :kconfig:option:`CONFIG_BT_MESH_ADV_BUF_COUNT` - Number of advertising buffers for local messages. Increase to improve performance, at cost of increased RAM usage.
+* :kconfig:option:`CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE` - Use a separate extended advertising set for GATT Server Advertising.
+* :kconfig:option:`CONFIG_BT_MESH_RELAY_ADV_SETS` - Maximum of simultaneous relay message support.
+* :kconfig:option:`CONFIG_BT_MESH_ADV_EXT_RELAY_USING_MAIN_ADV_SET` - Use the main advertising set to relay messages.
+
 Additional configuration options
 ================================
 
 This section lists additional configuration options that can be used to configure behavior and performance of Bluetooth mesh.
+Values given are only suggestions, and should be fitted to each individual application.
 
+* :kconfig:option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE` - Default system workqueue stack size maybe low and likely need to be increased.
+* :kconfig:option:`CONFIG_MAIN_STACK_SIZE` - Default initialization and main thread stack size maybe low and likely need to be increased.
+* :kconfig:option:`CONFIG_HWINFO` - See :ref:`bt_mesh_dk_prov`, chapter UUID.
+* :kconfig:option:`CONFIG_PM_SINGLE_IMAGE` - Use the Partition Manager for single image builds. See :ref:`partition_manager`
+* :kconfig:option:`CONFIG_PM_PARTITION_SIZE_SETTINGS_STORAGE` - Size of the partition used for settings storage. This should most likely be increased.
 * :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` - Enables partial erase on supported hardware platforms.
   Partial erase allows the flash page erase operation to be split into several small chunks preventing longer CPU stalls.
   This improves responsiveness of a mesh node during the defragmentation of storage areas used by the settings subsystem.
+* :kconfig:option:`CONFIG_DK_LIBRARY` - Button and LED Library for the nRF5x development kits. Should be added if nRF5x development kits are used.
+
+Logging
+-------
+
+* :kconfig:option:`CONFIG_NCS_SAMPLE_DEFAULTS` - See :ref:`ug_logging_zephyr`
+* :kconfig:option:`CONFIG_LOG_MODE_DEFERRED` - Recommended to avoid slowing mesh message processing.
+  Helps improve LPN power consumption when friendship is established.
+* :kconfig:option:`CONFIG_LOG_BUFFER_SIZE` - Increase to avoid missing logs when complex protocol or functionality issue.
+* :kconfig:option:`CONFIG_LOG_PROCESS_THREAD_SLEEP_MS` - Decrease to flush logs more quickly.
+
+GATT Proxy performance
+----------------------
+
+* These options allow fitting full relayed mesh advertiser frame into single Link Layer payload.
```

Comment:
Probably it is worth to mention that with this option they are only compatible with Bluetooth v4.2 devices and higher.